frontend_tests/node_tests/compose_fade.js

```diff
@@ -26,11 +26,11 @@ var bob = {
     full_name: 'Bob',
 };
 
-people.add(me);
+people.add_in_realm(me);
```

Comment:
Reason of change is: `add` function was only adding user in all people dict, where as `add_in_realm` function adds user in active people dict, which is used for condition in function `would_receive_message`.